goss-testing/scripts/check_weave.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+#
+# MIT License
+#
+# (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+
+set -exo pipefail
+
+TMPFILE=$(mktemp)
+
+WEAVE_POD=$(kubectl -n kube-system get pods -l name=weave-net -o jsonpath='{.items[*].metadata.name}' | awk '{ print $1 }')
+kubectl -n kube-system exec "${WEAVE_POD}" -c weave -- /home/weave/weave --local status ipam > "$TMPFILE"
```

Comment:
This command is not right.   You want to check `status connections` not `status ipam`.

Reply:
I see now that you are running the `status connections` below but not checking specifically for `seeded by different peers`.   You want to combine the command you have below with the check you have above.   And do it on all nodes/pods.

I think the tests we had in goss was never correct.

Reply:
Ok, I think I understand the request here. Let me make the changes and see if it's what you have in mind.

Reply:
I have made the changes and tested them. Ready for a re-review